keyboards/ikeygai/albatross/readme.md

```diff
@@ -0,0 +1,33 @@
+# Ikeygai Designs Albatross QMK firmware
+
+![ALBT](https://i.imgur.com/3hzbqkGh.jpg)
+
+Albatross is a TKL keyboard made by Ikeygai Designs.
+
+* Keyboard Maintainer: [Gondolindrim](https://github.com/Gondolindrim)
+* Hardware Supported: proprietary PCB using STM32F401 MCU
+* Hardware Availability: group buy for the Albatross will begin on february of 2024.
+
+## How to flash
+
+### Enter bootloader
+
+The DFU state in the bootloader can be accessed in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (top left key or Escape) and plug in the keyboard
+* **Physical reset button**: press the button on the back of the PCB, and hold it for five seconds
+* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available (Escape key on the default keymap)
+
+## Compile firmware
```

Comment:
I think you meant to have a third # here?

Reply:
Indeed I did. Fixed in [b015b1e](https://github.com/qmk/qmk_firmware/pull/22941/commits/b015b1e49800b79f6b1e88818ebc100649f7d5d2)